tests/test_experiment.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for EWFS violation computations."""
+
+import pytest
+import qiskit_aer
+from ewfs.experiment import run_experiment
+
+
+@pytest.mark.parametrize("backend", [qiskit_aer.Aer.get_backend("aer_simulator")])
+@pytest.mark.parametrize("charlie_sizes, debbie_sizes", [(range(1, 2), range(1, 2)), (range(1, 3), range(1, 2))])
+@pytest.mark.parametrize(
+    "settings",
+    [
+        [
+            ("peek", "reverse_1"),
+            ("peek", "reverse_2"),
+            ("reverse_2", "reverse_1"),
+            ("reverse_2", "reverse_2"),
+        ],
+        None,
+    ],
+)
+def test_random_stratgegy(backend, charlie_sizes, debbie_sizes, settings):
+    results = run_experiment(
+        shots=10_000,
```

Comment:
```suggestion
        shots=1_000,
```
Changing this from 10k to 1k improved test time on my laptop from 15 seconds to under 4 seconds. Is there a reason it needs to be 10k?

Reply:
Moved to #54